src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -69,6 +69,7 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
 
                 // get the target field names to be used by getRecord
                 this.fieldNames = getRecordFieldNames(this.formTemplate, this.fieldMappings);
+                this.initializeForm(this.formTemplate);
```

Comment:
@bdvorachek Did you check if the contact / account field values still display after adding this line?

Reply:
I've updated this to handle the case where we do have a donor record id present and need to load existing data. If there is one present, the initializeForm call should happen during the wiredGetRecord for the donor.

Reply:
I'm getting an exception, it looks like isEmpty() is being used in utilTemplateBuilder, but it was moved to utilCommon. It needs to be imported.

Uncaught (in promise) ReferenceError: LWC component's @wire target property or method threw an error during value provisioning. Original error:
[isEmpty is not defined]
    at Object.e.setRecordValuesOnTemplate (/lightning/n/modules/c/utilTemplateBuilder.js:4)